bundle-workflow/src/read_write_s3.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import boto3
+
+
+class Read_Write_Files:
+
+    # get_S3Objects:Used for getting the files from S3
+    # bucketname: Name of the bucket you want to get the file from
+    # RoleArn: Amazon Resource Name (ARN) of the role that you want to assume
+    # RoleSessionName: Name for the session
+    # bucket_bundle: Object path from where you want to get the bundle
+    # bucket_manifest: Object path from where you want to get the manifest
+    # path_tosave_Bundle: Path where you want to save the bundle at
+    # path_tosave_Manifest: Path where you want to save the manifest at
+    def __init__(self, RoleArn, RoleSessionName):
+        self.RoleArn = RoleArn
+        self.RoleSessionName = RoleSessionName
+
+    def get_S3Objects(self, bucketname, bucket_bundle, bucket_manifest, path_tosave_Bundle, path_tosave_Manifest):
+        sts_connection = boto3.client('sts')
+        acct_b = sts_connection.assume_role(
```

Comment:
Let's name it as `assumed_role` for clarity

Reply:
As the function name? because boto3 can only take assume_role() as the function

Reply:
Sorry if I was not clear before. Name `acct_b` as `assumed_role`